src/yb/tablet/tablet_bootstrap.cc

```diff
@@ -335,13 +352,16 @@ void ReplayState::DumpReplayStateToStrings(std::vector<std::string>* strings)  c
       pending_replicates.size(),
       OpIdToString(regular_stored_op_id),
       OpIdToString(intents_stored_op_id)));
+  int cnt = 1;
   if (num_entries_applied_to_rocksdb > 0) {
     strings->push_back(Substitute("Log entries applied to RocksDB: $0",
                                   num_entries_applied_to_rocksdb));
+    cnt++;
   }
   if (!pending_replicates.empty()) {
     strings->push_back(Substitute("Dumping REPLICATES ($0 items):", pending_replicates.size()));
-    AddEntriesToStrings(pending_replicates, strings);
+    cnt++;
+    AddEntriesToStrings(pending_replicates, strings, (limit-cnt)/2);
```

Comment:
Remove `cnt` altogether and just pass half_limit here.

Also or code style for expressions is `(limit - cnt) / 2`